helix-term/src/main.rs

```diff
@@ -61,9 +61,9 @@ ARGS:
 FLAGS:
     -h, --help                     Prints help information
     --tutor                        Loads the tutorial
-    --health [LANG]                Checks for potential errors in editor setup
-                                   If given, checks for config errors in language LANG
-    -g, --grammar {{fetch|build}}    Fetches or builds tree-sitter grammars listed in languages.toml
+    --health [CATEGORY]            Checks for potential errors in editor setup
+                                   CATEGORY can be a language or one of 'clipboard' or 'all'
```

Comment:
We could add a line saying "when omitted, CATEGORY is 'all'" just to make it extra clear. What do you think?

Reply:
Maybe it feels a bit weird to do `hx --health all` and not see the clipboard category?

Reply:
Hmm actually yeah, I think "all" might be better as "languages." I would expect "all" to act the same as not passing a category